src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,40 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { findTabId } from '../utils';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const currentTab = findTabId(tabs, selectedTabId);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={cn({
+                'is-active': tab.id === currentTab.id,
+              })}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (currentTab.id !== tab.id) {
+                    onTabSelected(tab);
+                  }
+                }}
```

Comment:
Let's create a handler for this instead of passing the anonim function to onclick